packages/framework/aqueduct/src/data-object-factories/pureDataObjectFactory.ts

```diff
@@ -179,14 +183,12 @@ export class PureDataObjectFactory<
 	}
 
 	/**
-	 * This is where we do data store setup.
-	 *
-	 * @param context - data store context used to load a data store runtime
+	 * {@inheritDoc @fluidframework/runtime-definitions#IFluidDataStoreFactory.instantiateDataStore}
 	 */
 	public async instantiateDataStore(
 		context: IFluidDataStoreContext,
 		existing: boolean,
-	): Promise<FluidDataStoreRuntime> {
+	): Promise<IFluidDataStoreChannel> {
```

Comment:
I realized the interface implementation chains are more complex than I thought... the concrete class implements `IFluidDataStoreChannel` but also `IFluidDataStoreRuntime` and `IFluidHandleContext`, so it could be that users of `PureDataObjectFactory` are currently leveraging members from any of those interfaces on the object returned by `PureDataObjectFactory.instantiateDataStore()`. Feels like there could be some cleanup we need to do here, but we probably want to be more careful about it.

Reply:
We can hold off on merging this for more reviewers to weigh in. These changes satisfy the requirements of the base interface, if consumers of aqueduct wish to do more, we should capture any additional requirements here.

Reply:
Since instantiateDataStore is from IFluidDataStoreFactory and that already specifies it as returning an IFluidDataStoreChannel I think you're fine with this change.  Theoretically someone could be invoking this off the class (rather than the interface) but I don't think that's something that really happens.